app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from __future__ import annotations
+
+
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    count_vaccinated_error = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (NotVaccinatedError, OutdatedVaccineError):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/207385897-2e6ae81f-c9ca-49ef-a521-f04e1f6b41eb.png)
fix this according to the task